frontend/frontend.go

```diff
@@ -19,6 +20,7 @@ type FrontendLLBBridge interface {
 	executor.Executor
 	Solve(ctx context.Context, req SolveRequest, sid string) (*Result, error)
 	ResolveImageConfig(ctx context.Context, ref string, opt llb.ResolveImageConfigOpt) (digest.Digest, []byte, error)
+	SessionManager() *session.Manager
```

Comment:
Why did you add this to bridge instead of a parameter to `Frontend.Solve` ?

Reply:
Updated: https://github.com/moby/buildkit/pull/1627/commits/03e1c199a37bb1ed58d118bce59d97eb630b0794
It was more obvious for me to add the bridge, but plumbing it through `Frontend.Solve` also works.